akka-docs/rst/java/http/routing-dsl/directives/alphabetically.rst

```diff
@@ -119,7 +119,7 @@ Directive                                   Description
                                             ``immutable.Seq[HttpHeader] ⇒ immutable.Seq[HttpHeader]`` function
 :ref:`-mapRouteResult-java-`                     Transforms the ``RouteResult`` with a ``RouteResult ⇒ RouteResult``
                                             function
-:ref:`-mapRouteResultFuture-java-`               Transforms the ``RouteResult`` future with a
+:ref:`-mapRouteResultStage-java-`               Transforms the ``RouteResult`` future with a
```

Comment:
There is those already here, but those are `ResultStage => CompletionStage[ResultStage]` while this is `CompletionStage[ResultStage] => CompletionStage[ResultStage]`